examples/doc_examples/quickstart/Makefile

```diff
@@ -0,0 +1,20 @@
+# This file is public domain, it can be freely copied without restrictions.
+# SPDX-License-Identifier: CC0-1.0
+
+# Makefile
+
+# defaults
+SIM ?= icarus
+TOPLEVEL_LANG = verilog
```

Comment:
Check out https://github.com/cocotb/cocotb/blob/master/tests/test_cases/test_sv_interface/Makefile. This stills runs the verilog test during the VHDL regression and can be overloaded to break using `make TOPLEVEL_LANG=vhdl`.

This only works for Questa and Riviera because they support both languages and GHDL skips tests when `TOPLEVEL_LANG` is set to `verilog`, which seems wrong to me... it should fail instead.

Reply:
I won't be able to make that change for the next two weeks, could you take over unless it can wait until that time?

Reply:
Yeah I can do that.

Reply:
It does not seem I have permission to push to your branch.